src/lib/prepareOas.ts

```diff
@@ -40,7 +40,17 @@ export default async function prepareOas(path: string, command: 'openapi' | 'val
 
     const fileFindingSpinner = ora({ text: 'Attempting to locate API definitions...', ...oraOptions() }).start();
 
-    const action = command === 'openapi' ? 'upload' : 'validate';
+    let action;
+    switch (command) {
+      case 'openapi':
+        action = 'upload';
+        break;
+      case 'openapi:reduce':
+        action = 'reduce';
+        break;
+      default:
+        action = command;
+    }
```

Comment:
once we migrate `validate` over to `openapi:validate` we should be able to just dynamically strip away the `openapi:` prefixes :chefs-kiss-cowboy: